HeroVault/HeroVault/Features/User/UserFavouritesScreen/UserFavouritesScreenViewModel.swift

```diff
@@ -46,13 +47,29 @@ class UserFavouritesScreenViewModel: ViewModelDelegate {
     func reloadView() {
     }
     
-    func createImage(marvelDataIndex: Int) -> String {
-        guard var imageName = (marvelDataList?[marvelDataIndex].thumbnail) else { return "" }
-        imageName.append("/portrait_incredible.jpg")
-        return imageName.convertToHttps()
+    struct CharacterImage {
+        let imageURL: String
```

Comment:
Maybe move this object outside the class

Reply:
what is this for? If it is a new model, please put it in it's own model file